mongomock/aggregate.py

```diff
@@ -942,14 +942,24 @@ def _handle_count_stage(in_collection, database, options):
     return [{options: len(in_collection)}]
 
 
+def _handle_facet_stage(in_collection, database, options):
+    out_collection_by_pipeline = {}
+    for pipeline_title, pipeline in options.items():
+        out_collection_by_pipeline[pipeline_title] = list(process_pipeline(in_collection,
```

Comment:
Please break after `process_pipeline(`
the arguments should all fit on one line after that.